crates/evm/evm/src/inspectors/stack.rs

```diff
@@ -836,28 +897,15 @@ impl<'a> Inspector<&mut dyn DatabaseExt> for InspectorStackRefMut<'a> {
     ) -> CreateOutcome {
         // Inner context calls with depth 0 are being dispatched as top-level calls with depth 1.
         // Avoid processing twice.
```

Comment:
is this comment still accurate here now that we changed to == 1?
and only check the 0 depth later